news/2018-09-17-GSoC2018.html

```diff
@@ -0,0 +1,222 @@
+                        </ul>
+                    </nav>
+                </div>
+            </div>
+
+            <div class="flc-toc-tocContainer toc"> </div>
+            <article id="content" class="floe-content floe-news-item">
+                <h2> News </h2>
+
+                <!-- BEGIN markup for news item -->
+                <h3>Google Summer of Code 2018</h3>
+                <time class="floe-date" datetime="2018-09-17">September 17, 2018</time>
+                <p>
+                    From March to August 2018, the Floe project participated in
+                    <a href="https://summerofcode.withgoogle.com/organizations/6584170814373888/#projects">Google Summer of Code (GSoC)</a>,
+                    a global program funded by Google which focuses on bringing student developers into open source
+                    software development. Floe team members mentored five students in a variety of projects, with the
+                    aim of expanding the Open Education Resources (OER) community and building connections with budding
+                    open source software developers.
+                </p><p>
+                    Interested students selected from
+                    <a href="https://wiki.fluidproject.org/x/CwAOCQ"> projects suggested by the Floe team </a> and
+                    submitted proposals based on these ideas. Once the five students were notified of their acceptance,
+                    they were encouraged to engage with the Inclusive Design community and learn more about the
+                    <a href="https://inclusivedesign.ca/">Inclusive Design Institute (IDI) </a>, the
+                    <a href="https://fluidproject.org/"> Fluid Project </a> and Floe. At the end of the coding period
+                    in early August, the students each prepared a final report summarizing their work and were
+                    invited to present at a
+                    <a href="https://wiki.fluidproject.org/display/fluid/Community+workshops"> Community Workshop </a> which was
+                    <a href="https://idrc.cachefly.net/wiki.fluidproject.org/videos/IDRC_CommunityWorkshop_GoogleSummerOfCode_StudentPresentations_2018-08-15.mp4">recorded on video</a>.
+                </p><p>
+                    The projects that were selected for GSoC this year are diverse in quite a few respects. From
+                    technical details like the programming language, framework and style to more obvious aspects like
+                    their methods of interaction and presentation. That said, they all endeavoured to improve
+                    accessibility in computing and aid learning. What follows is a brief description of each project
+                    and a link to its author’s final report.
+                </p>
+                <h3><a href="https://summerofcode.withgoogle.com/projects/#5411955049758720">Import external data to MyL3</a> by Jeevan Prakash</h3>
```

Comment:
I think all project titles should be an H4 instead of an H3 (H3 is article title, H4 are sections).